Documentation/operations/upgrade.rst

```diff
@@ -319,6 +319,15 @@ New Options
   routing. It must be set whenever running in direct routing mode with IPv6
   masquerading enabled.
 
+Removed Options
+~~~~~~~~~~~~~~~
+
+* The endpoint config option ``Conntrack`` was removed. The option was used
+  to disable the stateful connection tracking for the endpoint. However, many
+  Cilium features depend on the tracking. Therefore the option to disable
+  the connection tracking was removed. In addition, we deprecated
+  the ``disable-conntrack`` option and made it non-operational.
```

Comment:
Maybe mention that we intend to remove it in v1.13?

```suggestion
  the connection tracking was removed.
* Related to the above, option ``disable-conntrack`` for the agent has been
  deprecated and made non-operational. It will be removed in version 1.13.
```